doc/transceiver-monitor-hld.md

```diff
@@ -95,6 +95,23 @@ Xcvrd will call this API to wait for the sfp plug in/out event, following exampl
                  
 It's possible that when received the plug in/out event, the transceiver eeprom is not ready for reading, so need to give another try if first reading failed. 
 
+#### 1.3.1 Transceiver plug in/out event implementation on mlnx platform ####

```

Comment:
In above section, "default value 0 means never timeout" is not good. A blocking function with timeout zero normally means no wait at all.